services/calculate_analytics/analyses/content_analysis_2025_09_22/hashtags/submit_hashtag_analysis_prod.sh

```
@@ -0,0 +1,87 @@
+PYTHON_CMD="/projects/p32375/bluesky-research/services/calculate_analytics/analyses/content_analysis_2025_09_22/hashtags/main.py"
+
+echo "🏷️ Starting Production Hashtag Analysis"
+echo "======================================="
+echo "SLURM Job ID: $SLURM_JOB_ID"
+echo "Running python command: $PYTHON_CMD"
+echo "Memory: $SLURM_MEM_PER_NODE MB"
+echo "Time Limit: $SLURM_TIME_LIMIT"
+echo ""
+
+# Change to script directory
+cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)